### ISSUE TITLE
Weblet certificate expired

### ISSUE THREAD

**Issue by dxwc** [tootsuite/mastodon#10776]
https://weblate.joinmastodon.org/ linked from https://github.com/tootsuite/mastodon/blob/master/CONTRIBUTING.md shows expired certificate error on Firefox(66.0.5, linux)

![image](https://user-images.githubusercontent.com/22579884/57959748-34c0e480-78d3-11e9-85db-19da2b0f65cc.png)

Error text:

```
https://weblate.joinmastodon.org/

Peer’s Certificate has expired.

HTTP Strict Transport Security: false
HTTP Public Key Pinning: false

Certificate chain:

-----BEGIN CERTIFICATE-----
MIIFZjCCBE6gAwIBAgISAwTsPRjJQ2kzVBM/Ut9pCX4uMA0GCSqGSIb3DQEBCwUA
MEoxCzAJBgNVBAYTAlVTMRYwFAYDVQQKEw1MZXQncyBFbmNyeXB0MSMwIQYDVQQD
ExpMZXQncyBFbmNyeXB0IEF1dGhvcml0eSBYMzAeFw0xOTAyMTEwNDMwNTRaFw0x
OTA1MTIwNDMwNTRaMCMxITAfBgNVBAMTGHdlYmxhdGUuam9pbm1hc3RvZG9uLm9y
ZzCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBALtDjUZIxfCZJPY6ATk1
iih5ww2Wn2lSnu7x6lAgF+4jqHVOEx7+TaE6vN22c70BkdR3tu0+lMHg3JWquNLB
68ETHMNi5tvuSESYncQQAkYwCHe9wyjE5hhmVdP3BrXa9stPEteMkqTnnvIqX7pl
hiYhvpBmAAfXQcNeuzlyRdMvuqebBJcueh/sVvpi2ATF5v5vTFGUrvz/idOGCFzK
VHU01vLForWfoA1Tlo8LK0kKNQdjZLEE7YwwKaL+K6LSMKYxMEQBVCCBvR7fDHlS
pxQaP3Zzd5b/TjdtCcSD/6meFneT4A+N+9H1CKpmGmM/em+DmiQTpvMRLxpRsBlQ
my8CAwEAAaOCAmswggJnMA4GA1UdDwEB/wQEAwIFoDAdBgNVHSUEFjAUBggrBgEF
BQcDAQYIKwYBBQUHAwIwDAYDVR0TAQH/BAIwADAdBgNVHQ4EFgQU3NRzWbhhPOSV
9+xu88uTRKYYE0kwHwYDVR0jBBgwFoAUqEpqYwR93brm0Tm3pkVl7/Oo7KEwbwYI
KwYBBQUHAQEEYzBhMC4GCCsGAQUFBzABhiJodHRwOi8vb2NzcC5pbnQteDMubGV0
c2VuY3J5cHQub3JnMC8GCCsGAQUFBzAChiNodHRwOi8vY2VydC5pbnQteDMubGV0
c2VuY3J5cHQub3JnLzAjBgNVHREEHDAaghh3ZWJsYXRlLmpvaW5tYXN0b2Rvbi5v
cmcwTAYDVR0gBEUwQzAIBgZngQwBAgEwNwYLKwYBBAGC3xMBAQEwKDAmBggrBgEF
BQcCARYaaHR0cDovL2Nwcy5sZXRzZW5jcnlwdC5vcmcwggECBgorBgEEAdZ5AgQC
BIHzBIHwAO4AdQDiaUuuJujpQAnohhu2O4PUPuf+dIj7pI8okwGd3fHb/gAAAWjb
CW5NAAAEAwBGMEQCIAO1fJK3wPl4yA3fsSTmjDVD6+YCpmzaIQIfAW9DL3OXAiAo
bCv4n1C6RqhfnRiMYiBFW4CxVLxxHheGom/qEaewxgB1ACk8UZZUyDlluqpQ/FgH
1Ldvv1h6KXLcpMMM9OVFR/R4AAABaNsJbqEAAAQDAEYwRAIgZ0BaukmarAROLQhi
fqmnaGgubL3xoZ7kvdi9ah2XmsICIBXlMcVkXS9ESpFcfqiu/b2efZ34VYQuPIFQ
NzcwPMnwMA0GCSqGSIb3DQEBCwUAA4IBAQB00feIeI93JOK4OEv/No0xPClE6bJ4
wJ/ifEmtHgH8nXGk1WwRyyhMuOE3KUMncu9bCUR2vpynL2kxCtkplwMIbKGKsemo
RsK6UvAsjCsaxyPJU9cQbvJfAU4A3J3h55uwRUT06ZXQpZx9aIUqIqX7WRJdscza
1zDjb6sqpE8xkxCzoJZCxWODvAfyieTsUlN0xD8qzaSIXZ9ACFv5qTZDkRVSHwUz
Mn+H619OfrefSi2dKPTvXfYsKyr+v1O8CQX+XeZaL26tU52QfZMzH+CZIyBhYDHP
pLQ8XHlP8KZOdatEUNTBYSRZ5Y9S1cIcwG2EOzu4wUCUhbEPbyp8ksy8
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
MIIEkjCCA3qgAwIBAgIQCgFBQgAAAVOFc2oLheynCDANBgkqhkiG9w0BAQsFADA/
MSQwIgYDVQQKExtEaWdpdGFsIFNpZ25hdHVyZSBUcnVzdCBDby4xFzAVBgNVBAMT
DkRTVCBSb290IENBIFgzMB4XDTE2MDMxNzE2NDA0NloXDTIxMDMxNzE2NDA0Nlow
SjELMAkGA1UEBhMCVVMxFjAUBgNVBAoTDUxldCdzIEVuY3J5cHQxIzAhBgNVBAMT
GkxldCdzIEVuY3J5cHQgQXV0aG9yaXR5IFgzMIIBIjANBgkqhkiG9w0BAQEFAAOC
AQ8AMIIBCgKCAQEAnNMM8FrlLke3cl03g7NoYzDq1zUmGSXhvb418XCSL7e4S0EF
q6meNQhY7LEqxGiHC6PjdeTm86dicbp5gWAf15Gan/PQeGdxyGkOlZHP/uaZ6WA8
SMx+yk13EiSdRxta67nsHjcAHJyse6cF6s5K671B5TaYucv9bTyWaN8jKkKQDIZ0
Z8h/pZq4UmEUEz9l6YKHy9v6Dlb2honzhT+Xhq+w3Brvaw2VFn3EK6BlspkENnWA
a6xK8xuQSXgvopZPKiAlKQTGdMDQMc2PMTiVFrqoM7hD8bEfwzB/onkxEz0tNvjj
/PIzark5McWvxI0NHWQWM6r6hCm21AvA2H3DkwIDAQABo4IBfTCCAXkwEgYDVR0T
AQH/BAgwBgEB/wIBADAOBgNVHQ8BAf8EBAMCAYYwfwYIKwYBBQUHAQEEczBxMDIG
CCsGAQUFBzABhiZodHRwOi8vaXNyZy50cnVzdGlkLm9jc3AuaWRlbnRydXN0LmNv
bTA7BggrBgEFBQcwAoYvaHR0cDovL2FwcHMuaWRlbnRydXN0LmNvbS9yb290cy9k
c3Ryb290Y2F4My5wN2MwHwYDVR0jBBgwFoAUxKexpHsscfrb4UuQdf/EFWCFiRAw
VAYDVR0gBE0wSzAIBgZngQwBAgEwPwYLKwYBBAGC3xMBAQEwMDAuBggrBgEFBQcC
ARYiaHR0cDovL2Nwcy5yb290LXgxLmxldHNlbmNyeXB0Lm9yZzA8BgNVHR8ENTAz
MDGgL6AthitodHRwOi8vY3JsLmlkZW50cnVzdC5jb20vRFNUUk9PVENBWDNDUkwu
Y3JsMB0GA1UdDgQWBBSoSmpjBH3duubRObemRWXv86jsoTANBgkqhkiG9w0BAQsF
AAOCAQEA3TPXEfNjWDjdGBX7CVW+dla5cEilaUcne8IkCJLxWh9KEik3JHRRHGJo
uM2VcGfl96S8TihRzZvoroed6ti6WqEBmtzw3Wodatg+VyOeph4EYpr/1wXKtx8/
wApIvJSwtmVi4MFU5aMqrSDE6ea73Mj2tcMyo5jMd6jmeWUHK8so/joWUoHOUgwu
X4Po1QYz+3dszkDqMp4fklxBwXRsW10KXzPMTZ+sOPAveyxindmjkW8lGy+QsRlG
PfZ+G6Z6h7mjem0Y+iWlkYcV4PIWL1iwBi8saCbGS5jN2p8M+X+Q7UNKEkROb3N6
KOqkqm57TH2H3eDJAkSnh6/DNFu0Qg==
-----END CERTIFICATE-----
```

**Comment by unasuke** [comment]
The mastodon project no longer used weblate to translate UI sentences.

https://github.com/tootsuite/mastodon/issues/10385#issuecomment-477331393

So, I think CONTRIBUTING.md should be changed.

**Comment by TheKinrar** [comment]
Fixed in #10778 and certificate is no longer expired anyway.
